dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -25,85 +28,62 @@ def compareRecords (recordA,recordB, fields) :
         return True
 
     @classmethod
-    def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions):
-        """ Loads a table based on a csv
+    def cleanData(cls, data, model, fieldMap, fieldOptions):
+        """ Cleans up a dataframe that contains domain values
 
         Args:
-            filename: CSV to load
-            model: ORM object for table to be loaded
-            interface: interface to DB table is in
-            fieldMap: dict that maps columns of the csv to attributes of the ORM object
+            data = dataframe of domain values
+            fieldMap: dict that maps columns of the dataframe csv to our db columns
             fieldOptions: dict with keys of attribute names, value contains a dict with options for that attribute.
-                Current options are "pad_to_length" which if present will pad the field with leading zeros up to
-                specified length, and "skip_duplicate" which ignores subsequent lines that repeat values.
+                Current options are:
+                 "pad_to_length" which if present will pad the field with leading zeros up to
+                specified length
+                "skip_duplicate" which ignores subsequent lines that repeat values
+                "strip_commas" which removes commas
```

Comment:
Again, I know these options weren't introduced here, but this sounds like a great use case for a `namedtuple` rather than a dict